CHANGELOG.md

```diff
@@ -0,0 +1,27 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog][Keep a Changelog] and this project adheres to [Semantic Versioning][Semantic Versioning].
+
+## [Unreleased]
+
+
+---
+
+## [Released]
+## [0.10.0] - 2021-07-27
+### Changed
+- Updated to FCS 39
+- Updated to Fantomas 4.5.1
```

Comment:
We use Fantomas only for sample code - not sure if it's worth to mention it in the changelog

Reply:
ah, good catch